templates/default/django_settings.py.erb

```diff
@@ -41,142 +43,82 @@ LANGUAGE_CODE = 'en-us'
 
 SITE_ID = 1
 
+#Google analytics ID. Enter the full id, as in: UA-xxxxxx-x 
+GOOGLE_ID = 'UA-9880071-1'
```

Comment:
It's that way in production, too, actually, which was the source for this document.

The correct default value is None, of course.  What other values should we be setting in the template?